plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/internal/util/LayoutUtil.java

```diff
@@ -13,17 +13,13 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ******************************************************************************/
-package org.eclipse.glsp.api.utils;
+package org.eclipse.glsp.server.internal.util;
```

Comment:
Should Util classes really be internal? Especially this one that can be used to manipulate the model, which is probably what client applications should focus on

Reply:
Hm, not sure.. I think those utilities that are not made for being used in the main use cases should be internal to minimize what we have to be careful to change. But I agree, if one _might_ be useful, it is the LayoutUtil.